src/test/java/org/mockito/internal/stubbing/defaultanswers/ReturnsSmartNullsTest.java

```diff
@@ -64,7 +64,7 @@ public void should_print_the_parameters_when_calling_a_method_with_args() throws
 
         Foo smartNull = (Foo) answer.answer(invocationOf(Foo.class, "withArgs", "oompa", "lumpa"));
 
-        assertThat(smartNull.toString())
+        assertThat(smartNull + "")
```

Comment:
I proposed that change to the previous PR (that was merged in the meantime - preventing Pascal to amend it). `smartNull + ""` is more tricky, but provides a little bit better error message in case of `smartNull` being `null`. However, it is a matter of taste, so I will not insist on a change if you don't like it.